docs/about/overview.md

```diff
@@ -0,0 +1,50 @@
+# What is the MyST stack?
+
+This section provides a high-level overview of the main concepts and tools in the MyST ecosystem. See the [guiding principles of the MyST ecosystem](../guiding-principles.md) for some more high-level context.
+
+Here are a few key concepts in MyST:
+
+(myst-markdown)=
+
+**MyST Markdown**. MyST stands for "Markedly Structured Text". It is a flavor of [markdown](https://en.wikipedia.org/wiki/Markdown) that was designed to make it easy to write content that can be parsed by a [MyST Document Engine](#myst-engine). MyST Markdown syntax was created to support functionality in the MyST specification like extensions, roles and directives, and cross-references. It was originally created for Jupyter Book via the [MyST Parser for Sphinx](https://myst-parser.readthedocs.io), but now serves as a standard markdown syntax across the MyST ecosystem.
+
+(myst-engine)=
+
+**A MyST Document Engine**. A MyST Document Engine parses text-based documents into [MyST AST](#myst-ast) according to the [MyST Specification](#myst-specification)[^myst]. For example, a MyST Document Engine might know how to transform a text file with [MyST Markdown](#myst-markdown) into a `.json` output that follows the [MyST Specification](#myst-spec). The official MyST Document Engine is documented at [mystmd.org/guide](https://mystmd.org/guide), and is maintained by the [Jupyter Book team](https://compass.jupyterbook.org)
+
+[^myst]: Most commonly, MyST engines parse [MyST Markdown](#myst-markdown). However, a MyST engine can parse any kind of text syntax as long as the _result_ is a [MyST AST](#myst-ast) that follows the [MyST specification](#myst-spec). This is why the [official MyST Document Engine](https://mystmd.org) can parse other kinds of markdown syntax, like some of Pandoc's syntax.
+
+(myst-specification)=
+
+**The MyST Specification**. A specification that describes the structure and function of MyST documents. It provides a framework for defining all of the kinds of content and metadata that a MyST document can contain. For example, the MyST specification defines how sections can contain paragraphs, how paragraphs can contain sentences, and how sentences can contain a "bolded" chunk of text. The specification is [defined at mystmd.org/spec](https://mystmd.org/spec).
```

Comment:
Does this describe MyST Markdown, as suggested in the last paragraph, or does it talk more about the structure for creating, e.g., a notebook document around MyST Markdown content?

If the first, this could perhaps be simplified a bit along the lines of "this is the official reference definition of the MyST Markdown syntax".

Reply:
The MyST spec is nominally two things: the AST, and the markup grammar. However, increasingly I _think_ we're moving to the AST being the spec, and the grammar being recognised independently. Would welcome the rest of @jupyter-book/core-team's view there. For me, the benefits are that we can tighten the concept of the AST being the "unit of exchange and manipulation", and avoid tying ourselves to a language that is ultimately just another markup language.[^nb]

[^nb]: In the sense that ultimately MyST could operate with something like djot, rst, qmd, or even LaTeX without adversely modifying the engine underneath.